x-pack/plugins/security_solution/public/flyout/shared/components/flyout_body.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FC } from 'react';
+import React, { memo } from 'react';
+import { EuiFlyoutBody, EuiPanel } from '@elastic/eui';
+
+interface FlyoutBodyProps extends React.ComponentProps<typeof EuiFlyoutBody> {
+  children: React.ReactNode;
+}
+
+/**
+ * Flyout body wrapper component with proper padding
```

Comment:
let's be a bit more precise and say `16px` padding or the default padding from EuiPanel